VerifyScreen

aahenry51 opened the pull request.

aahenry51 commented:
![image](https://user-images.githubusercontent.com/79045258/166075169-29844de8-022b-4f64-862e-5528d912024d.png)

aahenry51 commented:
![image](https://user-images.githubusercontent.com/79045258/166080094-ce5a5a33-87f3-47a5-9681-2ff034c1c9fe.png)

aahenry51 commented:
![image](https://user-images.githubusercontent.com/79045258/166168799-4ccabb6a-2ee6-4bef-b6bc-15569c604d0d.png)

Dulaya commented:
- Very important: Remove the nested Container. You only need one Container on the outside.
- You have a few type errors.
- Add background to container.
- Okay to remove `TVerifyScreenProps` fromt Verify.type.ts if you're not using it:

      export type TVerifyScreenProps = StackScreenProps<
        TAuthNavigationRoutes,
        EAuthNavigationRoutes.VERIFY
      >;

- Your Button should have rounder borderRadius.
- Make sure you use camelCase:
   - PinContainer --> pinContainer
- Add fontFamily and size to subtext, subtext2 and subtext2button.
- Change color.red to color.primary in subtext2button.
- Why is red the same as primary in color.ts? 
- Optional: 
   - Consider changing the name subtext2button to something like subText2Link because it's not a button.
   - Consider adding shading/shadow to the number inputs.
- Verification works very smooth. Good job. ✔️

aahenry51 commented:
> 

1.Done
2.Done
3.Done
4.Done
5. I set the button radius to lg
6.Done
7.Done
8. I'm not sure but I can change the color to a red one in color.ts
9. I changed the name and added an on press event

aahenry51 commented:
I'm getting this bottom white space 
![image](https://user-images.githubusercontent.com/79045258/166336371-45649829-bcb1-476d-954a-2e470e2dafb5.png)

lasharela commented on mobile/src/screens/verify/Verify.view.tsx:
```diff
@@ -0,0 +1,45 @@
+  OnPressResend,
+}) => {
+  return (
+    <Container>
```
header will be as prop

Dulaya reviewed (approved):
Thanks for fixing.
